lib/gitx/cli/review_command.rb

```diff
@@ -10,34 +10,34 @@ class ReviewCommand < BaseCommand
 
       BUMP_COMMENT_PREFIX = '[gitx] review bump :tada:'
       BUMP_COMMENT_FOOTER = <<-EOS.dedent
-  # Bump comments should include:
-  # * summary of what changed
-  #
-  # This footer will automatically be stripped from the created comment
+        # Bump comments should include:
+        # * summary of what changed
+        #
+        # This footer will automatically be stripped from the created comment
       EOS
       APPROVAL_COMMENT_PREFIX  = '[gitx] review approved :shipit:'
       APPROVAL_COMMENT_FOOTER = <<-EOS.dedent
-  # Approval comments can include:
-  # * feedback
-  # * post-release follow-up items
-  #
-  # This footer will automatically be stripped from the created comment
+        # Approval comments can include:
+        # * feedback
+        # * post-release follow-up items
+        #
+        # This footer will automatically be stripped from the created comment
       EOS
       REJECTION_COMMENT_PREFIX = '[gitx] review rejected'
       REJECTION_COMMENT_FOOTER = <<-EOS.dedent
-  # Rejection comments should include:
-  # * feedback for fixes required before approved
-  #
-  # This footer will automatically be stripped from the created comment
+        # Rejection comments should include:
+        # * feedback for fixes required before approved
+        #
+        # This footer will automatically be stripped from the created comment
       EOS
 
       desc 'review', 'Create or update a pull request on github'
-      method_option :description, :type => :string, :aliases => '-d', :desc => 'pull request description'
-      method_option :assignee, :type => :string, :aliases => '-a', :desc => 'pull request assignee'
-      method_option :open, :type => :boolean, :aliases => '-o', :desc => 'open the pull request in a web browser'
-      method_option :bump, :type => :boolean, :aliases => '-b', :desc => 'bump an existing pull request by posting a comment to re-review new changes'
-      method_option :approve, :type => :boolean, :desc => 'approve the pull request an post comment on pull request'
-      method_option :reject, :type => :boolean, :desc => 'reject the pull request an post comment on pull request'
+      method_option :description, type: :string, aliases: '-d', desc: 'pull request description'
```

Comment:
Line is too long. [96/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.